lib/std/fmt.zig

```diff
@@ -1915,6 +1915,54 @@ test "parseUnsigned" {
     try std.testing.expectError(error.InvalidCharacter, parseUnsigned(u8, "", 10));
 }
 
+/// Parses a number like '2G', '2Gi', or '2GiB'.
+pub fn parseIntSizeSuffix(buf: []const u8, radix: u8) ParseIntError!usize {
+    var without_b = buf;
+    if (mem.endsWith(u8, buf, "B")) without_b.len -= 1;
+    var without_i = without_b;
+    var base: usize = 1000;
+    if (mem.endsWith(u8, without_b, "i")) {
```

Comment:
`sno/if (mem.endsWith(u8, without_b, "i")) {/if (mem.endsWith(u8, without_i, "i")) {`

Reply:
Wouldn't that be inconsistent with the first case? On line 1921, it checks the slice defined before `without_b`, so I figured that this line should check the slice defined before `without_i`.